src/main/java/de/photon/anticheataddition/util/log/Log.java

```diff
@@ -69,7 +69,7 @@ public static void error(String message, Throwable thrown)
         INSTANCE.logger.log(Level.SEVERE, message, thrown);
     }
 
-    private Logger logger;
+    public Logger logger;
```

Comment:
As https://github.com/Photon-GitHub/AntiCheatAddition/pull/679/files#r2052949394 should be reverted, this no longer needs to be public